docs/user-guides/workspace-access/remote-desktops.md

```diff
@@ -1,70 +1,26 @@
 # Remote Desktops
 
-## VNC Desktop
+## RDP
 
-The common way to use remote desktops with Coder is through VNC.
+The most common way to get a GUI-based connection to a Windows workspace is by using Remote Desktop Protocol (RDP).
 
-![VNC Desktop in Coder](../../images/vnc-desktop.png)
-
-Workspace requirements:
-
-- VNC server (e.g. [tigervnc](https://tigervnc.org/))
-- VNC client (e.g. [novnc](https://novnc.com/info.html))
-
-Installation instructions vary depending on your workspace's operating system,
-platform, and build system.
-
-As a starting point, see the
-[enterprise-desktop](https://github.com/coder/images/tree/main/images/desktop)
-image. It can be used to provision a Dockerized workspace with the
-following software:
-
-- Ubuntu 24.04
-- XFCE Desktop
-- KasmVNC Server and Web Client
+<div class="tabs">
 
-## RDP Desktop
+### Desktop Client
 
 To use RDP with Coder, you'll need to install an
 [RDP client](https://docs.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-clients)
 on your local machine, and enable RDP on your workspace.
 
 <div class="tabs">
 
-### CLI
-
-Use the following command to forward the RDP port to your local machine:
-
-```console
-coder port-forward <workspace-name> --tcp 3399:3389
-```
-
-Then, connect to your workspace via RDP at `localhost:3399`.
-![windows-rdp](../../images/ides/windows_rdp_client.png)
-
-### RDP with Coder Desktop
-
-[Coder Desktop](../desktop/index.md)'s Coder Connect feature creates a connection to your workspaces in the background.
-There is no need for port forwarding when it is enabled.
+#### Coder Desktop
 
-Use your favorite RDP client to connect to `<workspace-name>.coder` instead of `localhost:3399`.
+[Coder Desktop](../desktop/index.md)'s **Coder Connect** feature creates a connection to your workspaces in the background. Use your favorite RDP client to connect to `<workspace-name>.coder`.
 
-> [!NOTE]
-> Some versions of Windows, including Windows Server 2022, do not communicate correctly over UDP
-> when using Coder Connect because they do not respect the maximum transmission unit (MTU) of the link.
-> When this happens, the RDP client will appear to connect, but displays a blank screen.
->
-> To avoid this error, Coder's [Windows RDP](https://registry.coder.com/modules/windows-rdp) module
-> [disables RDP over UDP automatically](https://github.com/coder/registry/blob/b58bfebcf3bcdcde4f06a183f92eb3e01842d270/registry/coder/modules/windows-rdp/powershell-installation-script.tftpl#L22).
->
-> To disable RDP over UDP, run the following in PowerShell:
->
-> ```powershell
-> New-ItemProperty -Path 'HKLM:\SOFTWARE\Policies\Microsoft\Windows NT\Terminal Services' -Name "SelectTransport" -Value 1 -PropertyType DWORD -Force
-> Restart-Service -Name "TermService" -Force
-> ```
+You can use the [RDP Desktop](https://registry.coder.com/modules/coder/local-windows-rdp) module to add a single-click button to open an RDP session in the browser.
```

Comment:
I suggest adding a screenshot or GIF here of what it looks like from the desktop client. Would significantly improve the visual identity of the first part of the page and show what the Desktop UX looks like.